cmake/cross-toolchain.cmake

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+# Specify the sysroot
+set(sdk_target_sysroot "/opt/poky/3.1.7/sysroots/aarch64-poky-linux")
+set(CMAKE_SYSROOT ${sdk_target_sysroot})
+
+# Specify the cross compiler
+set(toolchain "/opt/poky/3.1.7/sysroots/x86_64-pokysdk-linux/usr/bin/aarch64-poky-linux/")
+set(toolchain_prefix "aarch64-poky-linux-")
+set(triple aarch64-poky-linux)
+set(CMAKE_C_COMPILER_TARGET ${triple})
+set(CMAKE_C_COMPILER ${toolchain}${toolchain_prefix}clang)
```

Comment:
${toolchain}${toolchain_prefix} -> ${toolchain}/${toolchain_prefix}

Reply:
By the way, why do think you need `${toolchain}`? The path to cross-compiler is usually already added by developers. I think it is not nessesary. Also, `${toolchain_prefix}` is also the same because `CMAKE_C_COMPILER_TARGET` is added in this file.